src/components/graph/graph.helper.js

```diff
@@ -360,7 +362,7 @@ function initializeGraphState({ data, id, config }, state) {
     let links = _initializeLinks(graph.links); // matrix of graph connections
     const { nodes: d3Nodes, links: d3Links } = graph;
     const formatedId = id.replace(/ /g, '_');
-    const simulation = _createForceSimulation(newConfig.width, newConfig.height);
+    const simulation = _createForceSimulation(newConfig.width, newConfig.height, newConfig.d3.gravity);
```

Comment:
Can you please check whether some is wrong with this line? Tests for `graph.helper` are breaking in this property fetch, it seems that `newConfig.d3` is undefined. Check [this link](https://travis-ci.org/danielcaldas/react-d3-graph/builds/426176598?utm_source=github_status&utm_medium=notification) for more detail.

Reply:
Yes, so in those particular unit tests I was checking for API compatibility I wasn't particularly interested in checking whether `config` was a string or other data type.